assets/locale/zh_CH.po

```diff
@@ -131,10 +131,10 @@ msgid "Launch Settings"
 msgstr "启动选项"
 
 msgid "Command"
-msgstr "命令"
+msgstr "指令"
```

Comment:
I think it would be more appropriate to use "命令" rather than "指令" to describe command line. 
"指令" is usually a translation of "instruction" in Chinese.

Reply:
So it is better to keep the original 命令

Reply:
> So it is better to keep the original 命令

yes